<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>180</y>
      <width>160</width>
      <height>146</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="btnNuevoProyecto">
       <property name="text">
        <string>Nuevo Proyecto...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnConfirmar">
       <property name="text">
        <string>Confirmar Acci√≥n...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCentroMens">
       <property name="text">
        <string>Centro de mensajes...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnPref">
       <property name="text">
        <string>Preferencias de usuario...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bntExport">
       <property name="text">
        <string>Exportar informe...</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>568</width>
     <height>33</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHerramientas">
    <property name="title">
     <string>Herramientas</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuHerramientas"/>
   <addaction name="menuAyuda"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
